demo.py

```diff
@@ -21,7 +21,8 @@ def find_coeffs(pa, pb):
 im = Image.new("RGBA", (1000, 1000), bgcolor).convert("RGBA")
 win = Image.open(win_path).convert("RGBA")
 
-terrace_fname = macintoshplus.windows[7]
+random_picture = numpy.random.choice(macintoshplus.windows)
```

Comment:
https://github.com/rickyhan/macintoshplus/blob/master/macintoshplus.py#L188

Instead of using a random seed, can you refactor the code a bit and use the supplied seed.